src/containers/Page.js

```diff
@@ -26,12 +25,44 @@ export default class Page extends PureComponent {
     }
   }
 
+  joinGame = () => this.props.joinGame(this.props.game.id)
+
+  makeMove = (userid, attacktype) => {
+    const {game, updateGame} = this.props
+    updateGame(game.id, attacktype)
+  }
+
+
   render() {
+    const {game} = this.props
     return (
+      <div>
+      {game === null &&
       <div className='game'>
-        <BattleArena background='forest' player1='fighter' player2='mage'/>
+        <BattleArena background='forest' player1='no' player2='no'/>
```

Comment:
Can't you get the players from redux state?

Reply:
yes i can and yes i will.